science/hacktv/Portfile

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+PortGroup           github 1.0
+
+github.setup        kesterlester hacktv 0.0.4 v
```

Comment:
I agree with @catap this should come from `fsphil hacktv` not your fork. You can remove the `github.tarball_from` and just use a commit hash and specify a (likely) date-based version in the Portfile.

Reply:
This is now done (in  199b2b38e0f7d34dd626218fb856dd3a62a86c07  ) but am still working on other things.